content/en/docs/security/_index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Security
+cascade:
+  vers: 0.115.1
```

Comment:
If this is the collector version then:

- The key should make that clear; e.g., use a name like `collector_vers`
- You'll need to modify the auto-version update script(s) to ensure that this parameter gets updated.

Reply:
Thank you! I was wondering how this shortcode worked. I think I updated the correct script, but if you could take a look, I'd appreciate it. 

For a straightforward copy-paste PR, I learned a lot! :smile: